src/css/claiming.less

```diff
@@ -522,16 +522,10 @@ form {
   padding-left: 34px;
 }
 
-#claiming-social-security #age-selector-response .next-steps li.next-step-one {
-  background-image: url('../images/step_1.png');
-}
-
-#claiming-social-security #age-selector-response .next-steps li.next-step-two {
-  background-image: url('../images/step_2.png');
-}
-
+#claiming-social-security #age-selector-response .next-steps li.next-step-one,
+#claiming-social-security #age-selector-response .next-steps li.next-step-two,
 #claiming-social-security #age-selector-response .next-steps li.next-step-three {
-  background-image: url('../images/step_3.png');
+  background-image: url('../images/step_check.png');
```

Comment:
We put pixel dimensions in the filename for some images. Not sure if you want that here or not.

Reply:
I kept the filename for the new image consistent with the filename formats in the older images (hence the underscore instead of the hyphen, which killed me a little 😄 ).